<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
background-color: qradialgradient(spread:repeat, cx:0.494, cy:0.226364, radius:1.255, fx:0.488682, fy:0.961, stop:0 rgba(55, 255, 25, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 225, 25, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
    </property>
    <property name="text">
     <string>Coffe Leaf Rust Severity Detector</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>581</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="shutdownButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Simplified Arabic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Shutdown
Device</string>
     </property>
    </widget>
    <widget class="QPushButton" name="imageProcButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Simplified Arabic</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Image
Processing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Simplified Arabic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Close
Program</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ledButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Simplified Arabic</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LED
ON/OFF</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
